src/main/java/org/team1540/robot2019/Hardware.java

```diff
@@ -229,17 +245,21 @@ public static void initCargoMech() {
         logger.info("Initializing cargo mech...");
         double start = RobotController.getFPGATime() / 1000.0; // getFPGATime returns microseconds
 
-        cargoRollerTop = createController(RobotMap.CARGO_ROLLER_TOP);
+        //        cargoRollerTopTalon.configPeakCurrentLimit(50);
+
+        ChickenTalon cargoRollerTop = new ChickenTalon(RobotMap.CARGO_ROLLER_TOP);
+        Hardware.cargoRollerTop = cargoRollerTop;
```

Comment:
Why qualified access?

Also should maybe check if that actually is a talon on the competition robot as I don't quite remember